ITensorGaussianMPS/test/gmps.jl

```diff
@@ -2,8 +2,9 @@ using ITensorGaussianMPS
 using ITensors
 using LinearAlgebra
 using Test
-
-const Pairing = ITensorGaussianMPS.ConservingNfParity
+import ITensorGaussianMPS.ConservesNf
+import ITensorGaussianMPS.ConservesNfParity
+const Pairing = ConservesNfParity
```

Comment:
Should be able to replace this with:
```julia
using ITensorGaussianMPS: ConservesNf, ConservesNfParity
```

Reply:
`import` let's you overload the type/function, while `using` just brings it into the namespace.